File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        num = max(candies)
        for i in range(0,len(candies),1):
            if candies[i]+extraCandies >= num:
                candies[i]=True
            else:
                candies[i]=False
        return candies
            
        

File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> arr = new ArrayList<>();
        int max = 0;
        for(int i=0 ; i<candies.length ; i++){
            if(max < candies[i]) max = candies[i];
        }
        for(int i=0 ; i<candies.length ; i++){
            if(candies[i] + extraCandies >= max)
                arr.add(true);
            else
                arr.add(false);
        }
        return arr;
    }
}